network.go

```diff
@@ -115,7 +116,7 @@ type CreateNetworkOptions struct {
 	Driver         string                 `json:"Driver" yaml:"Driver"`
 	IPAM           IPAMOptions            `json:"IPAM" yaml:"IPAM"`
 	Options        map[string]interface{} `json:"Options" yaml:"Options"`
-	Label          map[string]string      `json:"Labels" yaml:"Labels"`
+	Labels         map[string]string      `json:"Labels" yaml:"Labels"`
```

Comment:
Oh, I didn't catch this before :-(

I'll merge and hope that we don't break anyone.